test/unit/test_phonon.gd

```diff
@@ -234,10 +234,16 @@ func test_phonon_ua_exceed():
 	execute_strike(player1, player2, "uni_normal_assault", "uni_normal_assault", [], [], false, false, [], [], 0, [])
 	validate_positions(player1, 5, player2, 6)
 	validate_life(player1, 26, player2, 20)
-	
+
 
 func test_phonon_turning_reveal_nocopy():
 	position_players(player1, 3, player2, 6)
+	var copies_to_remove = []
+	for card in player1.hand:
```

Comment:
Even easier, you could just call discard_hand()

Reply:
oh true